plugin/src/shared/sc/plugin2021/util/GameRuleLogic.kt

```diff
@@ -204,7 +204,7 @@ object GameRuleLogic {
             if (throws) {
                 throw InvalidMoveException("Can't Skip on first round", SkipMove(gameState.currentColor))
             } else {
-                return MoveMistake.SKIP_FIRST_TURN
+                return SkipFirstTurn(gameState.currentColor)
             }
```

Comment:
Turn or Round? Please be consistent.

Reply:
It's effectively the same, since the first round is the first turn of each color.
I replaced this with MoveMistake's message, though.